docs/source/index.rst

```diff
@@ -153,9 +153,8 @@ Link to Cleanlab Studio docs: `help.cleanlab.ai <https://help.cleanlab.ai/>`_
 
    Datalab Tutorials <tutorials/datalab/index>
    Workflows of Data-Centric AI <tutorials/indepth_overview>
+   CleanLearning Tutorials <tutorials/clean_learning/index>
    Image Classification <tutorials/image>
-   Text Classification <tutorials/text>
-   Tabular Classification <tutorials/tabular>
    Audio Classification <tutorials/audio>
```

Comment:
let's move the image classification and audio classification tutorials into Datalab Tutorials?

it's otherwise a bit odd that these two appear in the main side bar, but no text or tabular tutorial appears in the main sidebar (makes it seem like the package only does image or audio)